### PR TITLE
feat: Added avatar and badge component

### PR DESCRIPTION
**Pull request by xii-vi**
# Avatar

Screenshot (19)

# Badge
Screenshot (20)

### CODE REVIEW

File: stylesheet/avatar.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    bottom: 0px;
+    right: 0px;
+}
+
+.dnd{
+    background-color: #ff0000;
+}
+.online{
+    background-color: #32b643;
+}
```
Comment (haarsh24) [inline stylesheet/avatar.css]: you can declare these color variables at top and use it here

File: stylesheet/badge.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.badge{
+    position: absolute;
+    background-color: #f44336;
+    color: #fff;
```
Comment (haarsh24) [inline stylesheet/badge.css]: same here

File: pages/avatar.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                    <img class="avatar avatar-xl" alt="avatar-demo" src="/assets/avatar/avatar-1.png">
+                    <img class="avatar avatar-lg" alt="avatar-demo" src="/assets/avatar/avatar-2.png">
+                    <img class="avatar avatar-std" alt="avatar-demo" src="/assets/avatar/avatar-3.png">
+                    <img class="avatar avatar-sm" alt="avatar-demo" src="/assets/avatar/avatar-4.png">
+                </div>
+                <p>The avatar component has 4 sizes, <span class="highlights p-1">avatar-xl</span>, <span
+                        class="highlights p-1">avatar-lg</span>,
+                    <span class="highlights p-1">avatar-std</span> and <span class="highlights p-1">avatar-sm</span>
+                </p>
+                <iframe
```
Comment (haarsh24) [inline pages/avatar.html]: you can use a good class name instead of a highlight

**Review by haarsh24 — APPROVED** [review]
some cosmetic changes.

Rest all looks good

File: stylesheet/avatar.css (hunk 1 above)
Comment (xii-vi) [reply stylesheet/avatar.css]: _Suggested changes are done._

File: pages/avatar.html (hunk 3 above)
Comment (xii-vi) [reply pages/avatar.html]: _Suggested changes are done._

File: stylesheet/badge.css (hunk 2 above)
Comment (xii-vi) [reply stylesheet/badge.css]: _Suggested changes are done._

File: pages/avatar.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>RA-One | Avatar</title>
```
Comment (Mohasin-Haque) [inline pages/avatar.html]: You can add meta description to personalise more.. 
LIke
<meta name="author" content="your name">

File: pages/avatar.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    <link rel="stylesheet" href="/stylesheet/main.css">
+    <link rel="stylesheet" href="/RA-One.css">
```
Comment (Mohasin-Haque) [inline pages/avatar.html]: you can import this in main css

File: pages/badge.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Mohasin-Haque) [inline pages/badge.html:6]: same can be done here too

File: pages/badge.html [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+                <span>RA-One</span>
+            </div>
+        </nav>
+        <div class="grid-20-60-20-layout">
+            <aside class="aside-container p-4">
+                <div>
+                    <p>Getting Started</p>
+                    <a class="p-4 flex" href="/pages/how-to-install.html">Usage</a>
+                    <p>Components</p>
+                    <a class="p-4 flex" href="#">Alert</a>
```
Comment (Mohasin-Haque) [inline pages/badge.html]: isnt it incorrect for this page?

File: stylesheet/badge.css [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    bottom: 0;
+    right: 0;
+    font-size: 60%;
+    font-style: var(--font);
+    width: 25%;
+    height: 25%;
+    font-weight: 600;
+    min-width: 13px;
+    min-height: 13px;
+    display: flex;
```
Comment (Mohasin-Haque) [inline stylesheet/badge.css]: Why not used relative units?

**Review by Mohasin-Haque — COMMENTED** [review]
Looks good but please try to run formatter and remove blank spaces for sanity. also go through the comments,

File: pages/avatar.html (hunk 4 above)
Comment (xii-vi) [reply pages/avatar.html]: _Suggested changes are done._

File: pages/avatar.html (hunk 5 above)
Comment (xii-vi) [reply pages/avatar.html]: _Suggested changes are done._

File: pages/badge.html (hunk 6 above)
Comment (xii-vi) [reply pages/badge.html:6]: _Suggested changes are done._

File: pages/badge.html (hunk 7 above)
Comment (xii-vi) [reply pages/badge.html]: _It gives the page name in the URL and It would made more sense in my opinion_

File: stylesheet/badge.css (hunk 8 above)
Comment (xii-vi) [reply stylesheet/badge.css]: _Suggested changes are done._

**Review by Mohasin-Haque — APPROVED** [review]
Now  the code is good. You can merge it.

**Pull request merged**